cypress/support/step-definitions/numeral-date-steps.js

```diff
@@ -40,3 +44,7 @@ Then('Date format in {string} input is set to {word}', (position, text) => {
 Then('Warning validation is visible', () => {
   icon().should('have.attr', 'type', 'warning').and('be.visible');
 });
+
+Then('Warning should not be visible', () => {
+  icon().should('not.be.visible');
```

Comment:
we need to precise which exactly icon should not be visible (warning in our case)